src/incubator/toast/index.tsx

```diff
@@ -165,8 +168,8 @@ const Toast = (props: PropsWithChildren<ToastProps>) => {
   };
 
   const renderIcon = () => {
-    const tintColor = iconColor;
-    return <Icon source={icon} resizeMode={'contain'} style={styles.icon} tintColor={tintColor}/>;
+    const tintColor = toastPreset.iconColor;
+    return <Icon source={toastPreset.icon} resizeMode={'contain'} style={styles.icon} tintColor={tintColor}/>;
```

Comment:
Are you keeping the `const tintColor` for readability?

Reply:
Nope (: 
Forgot it when I refactored the code. Fixed!